docs/pages/usage/proving.md

```diff
@@ -0,0 +1,142 @@
+# Running Stone Prover
+
+This guide provides step-by-step instructions on how to use the Stone Prover to create and verify proofs for Cairo programs. Stone Prover is a tool developed by StarkWare to facilitate proof generation and verification in the Cairo programming environment. This guide is based on the official [Stone Prover documentation](https://github.com/starkware-libs/stone-prover?tab=readme-ov-file#creating-and-verifying-a-proof-of-a-cairo-program).
+
+## Creating and Verifying a Proof
+
+To create and verify a proof using Stone Prover, follow these steps:
+
+#### 1. Clone the Stone Prover repository:
+
+```bash
+git clone https://github.com/starkware-libs/stone-prover.git
+cd stone-prover
+```
+
+#### 2. Build the prover
+
+Use Bazel to build the Stone Prover executable:
+
+```bash
+bazel build //src/main/prover:stone_prover
+```
+
+#### 3. Run the prover to generate a proof:
+
+Execute the prover with the necessary input arguments to generate a proof:
+
+```bash
+bazel-bin/src/main/prover/stone_prover \
+  --out_file=example_proof.json \
+  --program_name=fibonacci \
+  --parameter_file=src/main/prover/cpu_air_params.json \
+  --prover_config_file=src/main/prover/cpu_air_prover_config.json \
+  --public_input_file=src/main/prover/example_public_input.json \
+  --private_input_file=src/main/prover/example_private_input.json \
+  --air_file=src/main/prover/cpu_air.cairo \
+  --log_file=example_logs.txt
+```
+
+#### 4. Verify the generated proof:
+
+Similarly, you can verify proofs within the Docker container:
+
+```bash
+bazel-bin/src/main/prover/stone_prover \
+  --proof_file=example_proof.json \
+  --parameter_file=src/main/prover/cpu_air_params.json \
+  --public_input_file=src/main/prover/example_public_input.json \
+  --prover_config_file=src/main/prover/cpu_air_prover_config.json \
+  --air_file=src/main/prover/cpu_air.cairo
+```
+
+This command verifies the proof located at `/data/example_proof.json`
+
+## Input Arguments Explanation
+
+- `--out_file`: Specifies the output file for the generated proof.
+- `--program_name`: The name of the program being proved.
+- `--parameter_file`: File containing the proving system parameters.
+- `--prover_config_file`: Configuration file for the prover.
+- `--public_input_file`: JSON file with public inputs for the proof.
+- `--private_input_file`: JSON file with private inputs for the proof.
+- `--air_file`: The AIR (Algebraic Intermediate Representation) file for the CPU.
+- `--log_file`: File to store the prover logs.
+- `--proof_file`: The proof file to be verified (used in verification step).
+
+## Docker-based Setup
```

Comment:
Same here, please, try https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#creating-and-verifying-a-test-proof-using-binaries